examples/marvel.js

```diff
@@ -1,347 +0,0 @@
-(function() {
-    "use strict";
```

Comment:
Unless you feel like updating this one (heavily outdated), I prefer to delete it for now.